src/SFA.DAS.Rofjaa.Application/Agencies/Handlers/AgencyCommand.cs

```diff
@@ -0,0 +1,11 @@
+﻿using MediatR;
+
+namespace SFA.DAS.Rofjaa.Application.Agencies.Handlers
+{
+    public class AgencyCommand : IRequest<Unit>
+    {
+        public AgencyCommand()

```

Comment:
I wasn't clear on what this was, other than being used to register MediatR (in which case we could use another query and remove this?)